src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts

```diff
@@ -138,12 +141,22 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
   }
 
   showProgressDetail(target: HTMLElement, userProgress: TeamUserProgress, row: DataRow, col: DataColumn): void {
+    if (!this.itemData) {
+      throw new Error('Unexpected: Missed item data');
+    }
     if (!this.group) {
       throw new Error('Unexpected: Missed group');
     }
     this.progressOverlay = {
       target,
       progress: userProgress,
+      ...(this.currentFilter !== 'Groups' && col.type === 'Task' ? {
+        taskDetails: {
```

Comment:
`taskDetails` is not very clear (especially if you mix it `itemRoute`)... call it `colItem`maybe (I don't think you need to add "details" in the naming here.